src/main/java/chalmers/pimp/controller/IController.java

```diff
@@ -33,6 +33,11 @@
    */
   void selectRectangleTool();
 
+  /**
+   * Sets the doodle tool as the selectedTool in model
+   */
```

Comment:
Just a little note, try to avoid mentioning the actual names of fields in an implementation in documentation. Field names are prone to being changed.